### ISSUE TITLE
Some missing tags not being reported

### ISSUE THREAD

**Issue by iimog** [ThinkR-open/checkhelper#92]
I used `checkhelper::find_missing_tags()` to find missing `\value` tags in the Rd files of [`gggenomes`](https://thackl.github.io/gggenomes/) after CRAN rejected the package, because of this reason.
Some missing `\value` tags were reported by `checkhelper` and I fixed all of them. I re-checked until I saw the message:
```
Good! There is no missing or empty return value for exported functions
Good! There is no missing `@export` or `@noRd` in your documentation
```

After re-submitting to CRAN, we were rejected again, because there were still `\value` tags missing. I tried both the release version of `checkhelper` and the latest development version, both don't report the missing `\value` tags.
To reproduce, you can run:
```bash
git clone --depth 1 https://github.com/thackl/gggenomes -b missing-value-tags
cd gggenomes
R -s -e "mt <- checkhelper::find_missing_tags()"
```
and you will see
```
ℹ Loading gggenomes
Loading required package: ggplot2
Good! There is no missing or empty return value for exported functions
Good! There is no missing `@export` or `@noRd` in your documentation
ℹ Loading gggenomes
```
However, CRAN reports the following:
```
Missing Rd-tags:
      add_seqs.Rd: \value
      add_tracks.Rd: \value
      as_links.Rd: \value
      check_strand.Rd: \value
      dim.gggenomes_layout.Rd: \value
      drop_layout.Rd: \value
      flip.Rd: \value
      ggplot.gggenomes_layout.Rd: \value
      layout.Rd: \value
      strand_chr.Rd: \value
      strand_int.Rd: \value
      strand_lgl.Rd: \value
      track_ids.Rd: \value
```
And indeed, the `add_seqs.Rd` file, does not contain a `\value` tag. And so do the other files, reported by CRAN. Do you have an explanation, why these cases are not reported by `checkhelper`?

Sidenote: this code: https://stat.ethz.ch/pipermail/r-package-devel/2021q2/007106.html did report some, but also not all the remaining missing tags.
